app/views/editor/verifier/VerifierTest.js

```diff
@@ -115,13 +114,33 @@ module.exports = (VerifierTest = class VerifierTest extends CocoClass {
   }
 
   setupGod () {
-    this.god.setLevel(this.level.serialize({ supermodel: this.supermodel, session: this.session, otherSession: null, headless: true, sessionless: false }))
+    const level = this.level.serialize({ supermodel: this.supermodel, session: this.session, otherSession: null, headless: true, sessionless: false })
+    const hero = this.world.getThangByID('Hero Placeholder')
+    if (this.checkClampedProperties) {
+      level.constrainHeroHealth = true
+      if (!this.clampedProperties.inited) {
+        this.clampedProperties = {
+          inited: true,
+          maxHealth: { current: 1, lower: 1, upper: hero.maxHealth },
+          maxSpeed: { current: 1, lower: 1, upper: hero.maxSpeed },
+          attackDamage: { current: 1, lower: 1, upper: hero.attackDamage }
```

Comment:
Attack damage too high could also be a problem. We can start with lower as 4 and upper as 400 or something like this.